js/utils/resyncBlockchain.js

```diff
@@ -16,7 +16,7 @@ let server;
 
 // If you change this, be sure to change anywhere in the GUI you may have output how
 // long its unavailable.
-const resyncInactiveTime = 1000 * 60 * 60 * 1; // 1 hour
+const resyncInactiveTime = 1000 * 60;
```

Comment:
Since the server POST comes back immediately and independent of the actual blockchain resync process, I decided to leave a 1 minute inactive time in to at least limit the chance of concurrent server side calls being fired off.